### DISCUSSION TITLE
Email in Flex - Admin experience

### DISCUSSION THREAD

**Discussion by nicedisguise** [twilio-labs/paste#1395] [Office hours] [answered]
Beginning an ongoing discussion about the admin experience side of Email in Flex. The latest flow is [here](https://www.figma.com/proto/0XCJt2dfb7026AMMm6va7v/email-integration-in-flex-and-conversations-mapping?page-id=1261%3A3776&node-id=1710%3A26&viewport=245%2C617%2C0.11634460836648941&scaling=min-zoom) and I have scheduled a session for (May 6th) to go through it.

What I'll be looking for feedback on initially is the general flow and the use of certain components used.

Look forward to sharing

**Comment by nkrantz** [comment]
Thanks for sharing this flow @nicedisguise, we'll be sure to take a look and prepare some thoughts! Looking forward to discussing at next week's Office Hours.

**Comment by TheSisb** [comment, marked as answer]
Following up on today's the Office Hours. [Here's the link to the annotated recording](https://www.loom.com/share/accfd0ac8e554554a1b0154a6ca01988) (password is in slack). Thanks for coming!
